src/wallet/rpcwallet.cpp

```diff
@@ -345,7 +345,7 @@ UniValue listaddresses(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "[\n"
             "  {\n"
-            "    \"source\": \"imported|imported_watchonly|keypool|legacy_seed|mnemonic_seed\"\n"
+            "    \"source\": \"imported|imported_watchonly|keypool|legacy_seed|legacy_random|mnemonic_seed\"\n"
```

Comment:
We need to document the `unified` structure. This is what I think it should look like (per my other comments):
```json
"unified": [ -- each element in this list represents a set of diversified Unified Addresses derived from a single UFVK.
    {
        "account": 0,
        "addresses": [
            {
                "diversifier_index", 0,
                "receiver_types": [
                    "sapling",
                    ...
                ],
                "address": "..."
            },
            ...
        ]
    },
    ...
]
```

In particular:
- `account` and `diversifier_index` should be a JSON numbers (matching how we consume them in `z_getaddressforaccount`).
- `receiver_types` should be lowercase for consistent identification (the API should be machine-parseable first, human-parseable second).

We need to double-check all the key names above for consistency, at the same time as we do #5432 or #5534 (e.g. should `address` above be `unifiedaddress` like other APIs, or vice versa).

Reply:
Just to confirm, is this right?
```
"diversifier_index", 0,
```
should be:
```
"diversifier_index": 0,
```